src/openrct2/actions/PlacePeepSpawnAction.hpp

```diff
@@ -96,27 +97,26 @@ DEFINE_GAME_ACTION(PlacePeepSpawnAction, GAME_COMMAND_PLACE_PEEP_SPAWN, GameActi
     {
         auto res = std::make_unique<GameActionResult>();
         res->Expenditure = ExpenditureType::LandPurchase;
+        // TODO: Suspicious z/8 to transform Coords into Coords
         res->Position = CoordsXYZ{ _location.x, _location.y, _location.z / 8 };
 
         // If we have reached our max peep spawns, remove the oldest spawns
         while (gPeepSpawns.size() >= MAX_PEEP_SPAWNS)
         {
             auto oldestSpawn = gPeepSpawns.begin();
-            auto oldX = oldestSpawn->x;
-            auto oldY = oldestSpawn->y;
+            auto oldSpawnPos = CoordsXY{ oldestSpawn->x, oldestSpawn->y };
```

Comment:
Spawns are already typedef'd as CoordsXYZD - can't you just assign it directly to `oldSpawnPos`?

Reply:
I can, thanks for bringing it up.